scripts/reuse_skip_wrapper.py

```diff
@@ -26,7 +26,12 @@ def skip(file: str) -> bool:
     args = parser.parse_args()
 
     files = [f for f in args.files if not skip(f)]
+
+    # Check if reuse is installed in ~/.local/bin otherwise try to use python -m reuse
+    reuse_path = os.path.expanduser("~/.local/bin/reuse")
+    if not os.path.isfile(reuse_path):
+        reuse_path = "python -m reuse"
```

Comment:
I think what you want to do is check whether `reuse` is in the path with:
```py
shutil.which(executable) is not None
```
instead of checking whether it is in a specific path.

Reply:
You are of course right ;) Done in https://github.com/pulp-platform/Deeploy/pull/113/commits/c2dd40e1844c6326ce5aea1583706a3ea6db0a9d